schemas/app/BatchNumber.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "BatchNumber",
+    "label": "Batch Number",
+    "naming": "manual",
+    "fields": [
+        {
+            "fieldname": "name",
+            "fieldtype": "Data",
+            "create": true,
```

Comment:
Why have you added `"create"` to all these fields, `"create"` is meant to be used only with `Link` fields, it's mentioned in `schemas/types.ts`

Reply:
corrected